c10/util/flat_hash_map.h

```diff
@@ -389,15 +375,13 @@ class sherwood_v3_table : private EntryAlloc,
       return *this;
 
     clear();
-    if (AllocatorTraits::propagate_on_container_copy_assignment::value) {
+    if constexpr (AllocatorTraits::propagate_on_container_copy_assignment::
```

Comment:
This is also a vendored file and fixes should be added to the comment log at the top of this file to keep track of changes

Reply:
Added.